### PR TITLE
LOC CHECKIN | dotnet/winforms master | 20190131

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

**Comment by janbaltus** [comment]
Please wait with merge till 2/1. We neeed to update PR with fully localized xlf

**Comment by karelz** [comment]
@janbaltus marking as "NO MERGE" per your request

**Comment by janbaltus** [comment]
@karelz please merge now. PR is updated

**Comment by KlausLoeffelmann** [comment]
Hey Jan, good to see you here! The world is small, is it not? 😄 

Can you take a look at my review of the German part?
There was one wrong translation, and one that was kind of off.

Thanks! See you around!

**Comment by karelz** [comment]
"No merge" removed, I let merging on WinForms team members :) (pending code review changes based on Klaus's CR feedback)

File: src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.de.xlf [hunk 1]
```diff
@@ -4,117 +4,117 @@
...
       <trans-unit id="ColorEditorSystemTab">
         <source>System</source>
-        <target state="new">System</target>
+        <target state="translated">System</target>
         <note />
       </trans-unit>
       <trans-unit id="DockEditorAccName">
         <source>Dock Picker</source>
-        <target state="new">Dock Picker</target>
+        <target state="translated">Auswahl andocken</target>
```
Comment (zsd4yr) [inline src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.de.xlf]: per @KlausLoeffelmann should be Dockungsauswahl

File: src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.de.xlf [hunk 2]
```diff
@@ -4,117 +4,117 @@
...
       <trans-unit id="DockEditorBottomAccName">
         <source>Bottom</source>
-        <target state="new">Bottom</target>
+        <target state="translated">Unten</target>
         <note />
       </trans-unit>
       <trans-unit id="DockEditorFillAccName">
         <source>Fill</source>
-        <target state="new">Fill</target>
+        <target state="translated">Voll</target>
```
Comment (zsd4yr) [inline src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.de.xlf]: per @KlausLoeffelmann should be Füllend

**Comment by janbaltus** [comment]
Hi @KlausLoeffelmann, we are triaging your comments and we will implement them and export by end of this week.
Thank you,
Jan

File: src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.de.xlf (hunk 1 above)
Comment (lukaschal) [reply src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.de.xlf]: Fixed as suggested.

File: System.Windows.Forms.Design.Editors\src\Resources\xlf\SR.xlf
ID: 0; "DockEditorAccName"
SRC: Dock Picker
TGT: Dockungsauswahl

File: src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.de.xlf (hunk 2 above)
Comment (lukaschal) [reply src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.de.xlf]: Fixed as suggested.

File: System.Windows.Forms.Design.Editors\src\Resources\xlf\SR.xlf
ID: 0; "DockEditorFillAccName"
SRC: Fill
TGT: Füllend

**Pull request merged**